abb_irb1600_support/urdf/irb1600_10_145_macro.xacro

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro" name="irb1600_10_145">
+<xacro:macro name="abb_irb1600_10_145" params="prefix">
+  <!-- link list -->
+  <link name="${prefix}base_link">
+    <visual>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <geometry>
+        <mesh filename="package://abb_irb1600_support/meshes/irb1600/visual/base_link.stl"/>
+      </geometry>
+      <material name="abb_orange">
+        <color rgba="1 0.43 0 1"/>
+      </material>
```

Comment:
could you please use the shared colour / material definitions from `abb_resources` instead?

(same for all the other occurrences below)

Reply:
done, also for the 6/1.2 variant.